configure.ac

```diff
@@ -308,7 +308,7 @@ AC_MSG_RESULT([$LIBS])
 # Dependencies that themselves have a pkg-config file available.
 #
 PC_REQUIRES=""
-AS_IF([test "$pmix_hwloc_support_will_build" = "yes"],
+AS_IF([test "$pmix_hwloc_source" != "cobuild"],
       [PC_REQUIRES="$PC_REQUIRES hwloc"])
 AS_IF([test $pmix_libevent_support -eq 1],
```

Comment:
Wouldn't we have the same issue with the internal libevent? IIRC, the integration is pretty close to an exact duplicate.

Or does the libevent m4 code already take that into account when setting this variable? PMIx _requires_ libevent, so it isn't like pmix_libevent_support can be anything other than 1. I'm okay with leaving this question for separate resolution - just wondering why we don't have the same issue here.